setup.py

```diff
@@ -10,12 +10,21 @@
 from setuptools.command.build_ext import build_ext as build_ext_
 from setuptools.command.develop import develop as develop_
 from setuptools.command.install import install as install_
-from Cython.Compiler.Main import default_options
+
 import shutil
 import subprocess
 import sys
 import platform
 
+try:
+    from Cython.Compiler.Main import default_options
+except ImportError as e:
+    print("Cython is required to evaluate bitshuffle extensions")
+    # Install Cython via pip subprocess call
+    print ("Installing Cython...")
+    subprocess.call(["pip", "install", "Cython>=0.19"])
+    from Cython.Compiler.Main import default_options
```

Comment:
This is actively hostile behavior for a build script and should not be considered for inclusion in this repository.

The import fails, now you as a user know you need to install Cython in your environment. That may be through `conda`, `pip`, `pip install --user`, your Linux package manager, from source, or whatever other mechanism is interesting to you. A `setup.py` script making this choice for the user is not at all acceptable.